modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/main.es.js

```diff
@@ -85,74 +85,81 @@ class Form extends Component {
 				return editor;
 			}),
 			this._createEditor('descriptionEditor'),
-			this._getTranslationManager()
+			Liferay.componentReady('translationManager')
 		];
 
 		if (this.isFormBuilderView()) {
 			dependencies.push(this._getSettingsDDMForm());
 		}
 
-		Promise.all(dependencies).then(results => {
-			this._translationManager = results[2];
-
-			const translationManager = this._translationManager;
+		Promise.all(dependencies).then(
+			([
+				nameEditor,
+				descriptionEditor,
+				translationManager,
+				settingsDDMForm
```

Comment:
Just destructured the array here for clarity, rather than using positional `results`... this is as fragile as before, but at least is slightly more readable

Reply:
I totally agree, `results[2]` was very weird but when migrating I just left the code that hasn't been affect as is.